cartridges/bm_app_imgix_base/cartridge/experience/editors/custom/tileCustomView.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "IMGIX product view",
```

FYI, imgix should never be capitalized.

```suggestion
    "name": "imgix product view",
```